iothub/service/src/TransportType.cs

```diff
@@ -7,24 +7,18 @@ namespace Microsoft.Azure.Devices
     /// Transport types supported by MessagingClient, FileUploadNotificationProcessorClient and MessageFeedbackProcessorClient.
     /// </summary>
     /// <remarks>
-    /// Amqp and Amqp over WebSocket.
+    /// Only supports Amqp and Amqp over WebSocket.
     /// </remarks>
     public enum TransportType
```

Comment:
Can we adopt the naming scheme we came up with in the device client? See `IotHubClientTransportProtocol.cs`. For service I'd probably name it `IotHubTransportProtocol`.